xarray/tests/test_groupby.py

```diff
@@ -1490,9 +1504,13 @@ def test_resample(self):
         expected = DataArray.from_series(expected_)
         assert_identical(actual, expected)
 
-        with pytest.raises(ValueError, match=r"index must be monotonic"):
+        with pytest.raises(ValueError, match=r"Index must be monotonic"):
             array[[2, 0, 1]].resample(time="1D")
 
+        reverse = array.isel(time=slice(-1, None, -1))
+        with pytest.raises(ValueError):
```

Comment:
new test to make sure we error when trying to resample with a decreasing time index.